js/big-photo.js

```diff
@@ -34,13 +30,7 @@ function buttonClosePhotoKeydownHandler (evt) {
 function makeTextInfo (photo) {
   likesCount.textContent = photo.likes;
   socialCaption.textContent = photo.description;
-
-  socialCommentsCount.classList.add('hidden');
-  commentsCount.textContent = photo.comments.length;
-
-  commentsLoader.classList.add('hidden');
-
-  socialComments.appendChild(createFragmentComments(photo));
+  setInitialParametersComments(photo);
```

Comment:
```suggestion
  setComments(photo);
```

более чем достаточно